communicationmanager.go

```diff
@@ -137,13 +140,17 @@ func newCommunicationManager(cfg *config.Config) (cm *communicationManager, err
 		return cm, aoserrors.Wrap(err)
 	}
 
+	if cm.cryptoContext, err = cryptutils.NewCryptoContext(cfg.Crypt.CACert); err != nil {
+		return nil, aoserrors.Wrap(err)
+	}
+
```

Comment:
Also, the initialization of TPM device and PKCS11 library should be moved here from fcrypt.

Reply:
Done